### ISSUE TITLE
Last layer activation method used in training on Pascal VOC

### ISSUE THREAD

**Issue by Radhikadua123** [ramprs/grad-cam#17]
I got a small doubt regarding the last layer activation method used while training the model to generate the heatmaps. I assume you used softmax on top of the last layer logits for imagenet dataset. However, it is not mentioned anywhere how you trained the model with Pascal VOC dataset. Can you please provide some details on it? It will be of great help to me.
Thanks in advance!
